test/python/pulse/test_instructions.py

```diff
@@ -191,19 +191,28 @@ def test_default(self):
 class TestPlay(QiskitTestCase):
     """Play tests."""
 
+    def setUp(self):
```

Comment:
How are these changes related to the bug?

Reply:
Not directly related, but the current code will allow you to play pulses on `AcquireChannel`'s, which should be illegal.